receiver_custom.html

```diff
@@ -403,7 +406,19 @@ <h3 id= "error_description" class ="cast_text"></h3>
       // prepare the player load script
       var playerJs = document.createElement("script");
       playerJs.type = "text/javascript";
-      playerJs.src = OOYALA_PLAYER_URL + "?" + (debug? ("debug=true&" + v3Version) : v3Version);
+      playerJs.src = OOYALA_PLAYER_URL;
+      if (debug || v3Version) {
+        playerJs.src += "?";
+        if (debug) {
+          playerJs.src += "debug=true";
+          if (v3Version) {
+            playerJs.src += "&" + v3Version;
+          }
+        } else {
+          playerJs.src += v3Version;
+        }
+      }
```

Comment:
I disagree that it's excessive.  The "?" could be anywhere in the url.  If it is then we cannot put another "?" or it will fail to parse (two "?" means invalid url)